samples/dashboards/demo/vertical/demo.html

```diff
@@ -0,0 +1,117 @@
+<script src="https://code.highcharts.com/highcharts.js"></script>
+<script src="https://code.highcharts.com/dashboards/datagrid.js"></script>
+<script src="https://code.highcharts.com/dashboards/dashboards.js"></script>
+<script src="https://code.highcharts.com/dashboards/modules/layout.js"></script>
+<script src="https://code.highcharts.com/modules/accessibility.js"></script>
+
+<div id="container">
```

Comment:
Remove background color from #container so that background is white

Set max-width to 900 for shorter lines (better readability)

Reply:
What about the dark mode? Is this fine?
![image](https://github.com/user-attachments/assets/606268de-9667-47ee-8b35-040165a0361a)

Reply:
Hm.... Your darkmode looks different than mine.

<img width="1033" alt="Skjermbilde 2024-09-27 kl  08 55 23" src="https://github.com/user-attachments/assets/643b110e-20bd-4d97-959f-19d32e06acce">

Text should be white, like yours.

Reply:
How about the background in dark mode?

Reply:
In dark mode the text could be more white. On my screen it is a bit difficult to read.